summary_workflows/quantification/README.md

```diff
@@ -28,6 +28,12 @@ This README describes the APAeval **quantification** summary workflow. For a mor
 - `input_file`: output file from execution workflow in bed6 format
 - `gold standard`: bed6 file derived from 3'end sequencing on the same sample(s) as the RNA-seq data used in the challenge
 >NOTE: the gold standard file MUST be named in the format `[challenge].bed`, where `[challenge]` is specified in `challenges_ids` in `nextflow.config`. The extension `.bed` is hardcoded within [`compute_metrics.py`][metrics-py].
+- `windows` parameter is used to compute metrics for a list of window sizes.
+    - For running on OEB: the parameter is read from `nextflow.config`.
+- `genome_dir`: Directory to genome annotation in gtf format with 9 fields as specified [here](https://www.ensembl.org/info/website/upload/gff.html). The gtf is used for the relative PAS usage metric computation.
```

Comment:
Given the fact that we want to use Gencode annotations in APAeval, and are already having lot's of confusion with Gencode vs Ensembl I think it would make sense here to give a link to Gencode; https://www.gencodegenes.org/pages/data_format.html  
:smile: